account_analytic_salesperson/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This module ensures that analytic accounts are created for each sale order that
+is confirmed ands adds a Salesperson field on analytic accounts and analytic
+lines in order to keep track of the Salesperson that was defined on the sale
+order when the analytic account is created.
```

Comment:
But the saleperson is already in the account.invoice , why to take it from the sale order and nor from the invoice?

Reply:
We would like to create in one operation the analytic account and the salesperson linked to this account. In fact the sales person will be the same from SO and from Invoice.